internal/resource/configmap.go

```diff
@@ -81,15 +86,57 @@ func (builder *ServerConfigMapBuilder) Update(object runtime.Object) error {
 		if err := cfg.Append([]byte(defaultTLSConf)); err != nil {
 			return err
 		}
+		if builder.Instance.AdditionalPluginEnabled("rabbitmq_mqtt") {
+			if _, err := defaultSection.NewKey("mqtt.listeners.ssl.default", "8883"); err != nil {
+				return err
+			}
+		}
+		if builder.Instance.AdditionalPluginEnabled("rabbitmq_stomp") {
+			if _, err := defaultSection.NewKey("stomp.listeners.ssl.1", "61614"); err != nil {
+				return err
+			}
+		}
 	}
 
 	if builder.Instance.MutualTLSEnabled() {
-		if _, err := defaultSection.NewKey("ssl_options.cacertfile", "/etc/rabbitmq-tls/"+builder.Instance.Spec.TLS.CaCertName); err != nil {
+		if _, err := defaultSection.NewKey("ssl_options.cacertfile", "/etc/rabbitmq-tls/ca.crt"); err != nil {
 			return err
 		}
 		if _, err := defaultSection.NewKey("ssl_options.verify", "verify_peer"); err != nil {
 			return err
 		}
+
+		if _, err := defaultSection.NewKey("management.ssl.cacertfile", "/etc/rabbitmq-tls/ca.crt"); err != nil {
+			return err
+		}
+		if builder.Instance.AdditionalPluginEnabled("rabbitmq_web_mqtt") {
+			if _, err := defaultSection.NewKey("web_mqtt.ssl.port", "15676"); err != nil {
+				return err
+			}
+			if _, err := defaultSection.NewKey("web_mqtt.ssl.cacertfile", "/etc/rabbitmq-tls/ca.crt"); err != nil {
+				return err
+			}
+			if _, err := defaultSection.NewKey("web_mqtt.ssl.certfile", "/etc/rabbitmq-tls/tls.crt"); err != nil {
+				return err
+			}
+			if _, err := defaultSection.NewKey("web_mqtt.ssl.keyfile", "/etc/rabbitmq-tls/tls.key"); err != nil {
+				return err
+			}
+		}
+		if builder.Instance.AdditionalPluginEnabled("rabbitmq_web_stomp") {
+			if _, err := defaultSection.NewKey("web_stomp.ssl.port", "15673"); err != nil {
+				return err
+			}
+			if _, err := defaultSection.NewKey("web_stomp.ssl.cacertfile", "/etc/rabbitmq-tls/ca.crt"); err != nil {
```

Comment:
Should we extract some of these paths into variables?
`/etc/rabbitmq-tls/ca.crt` is used 4 times here.

Reply:
updated. Good suggestion!